From 8e759144e526e17946dc92e91f2dee947112b8ff Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@edality.by>
Date: Mon, 10 Jun 2019 19:19:03 +0300
Subject: iotedge: Update default configuration

Use a specific iotedge agent image until arm64 architecture is not yet
fully supported by azure.
---
 packages/apps/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/packages/apps/Makefile b/packages/apps/Makefile
index ba68887..f6ff143 100644
--- a/packages/apps/Makefile
+++ b/packages/apps/Makefile
@@ -666,13 +666,16 @@ azure-iotedge:
 ifeq ($(CONFIG_BUILD_AZURE_IOTEDGE), y)
 	mkdir -p $(FBDIR)/packages/apps/azure-iotedge && \
 	cd $(FBDIR)/packages/apps/azure-iotedge && \
+	rm -f $(azure_libiothsm_deb_name) $(azure_iotedge_deb_name) && \
 	wget $(azure_libiothsm_deb_url) && \
 	wget $(azure_iotedge_deb_url) && \
 	cp $(azure_libiothsm_deb_name) $(azure_iotedge_deb_name) $(RFSDIR)
 	sudo chroot $(RFSDIR) dpkg -i /$(azure_libiothsm_deb_name) && \
 	sudo chroot $(RFSDIR) rm -f /$(azure_libiothsm_deb_name)
 	sudo chroot $(RFSDIR) dpkg -i /$(azure_iotedge_deb_name) && \
-	sudo chroot $(RFSDIR) rm -f /$(azure_iotedge_deb_name)
+	sudo chroot $(RFSDIR) rm -f /$(azure_iotedge_deb_name) && \
+	sudo sed -i 's_^hostname: ".*_hostname: "trustbox"_' $(RFSDIR)/etc/iotedge/config.yaml && \
+	sudo sed -i 's_image: ".*_image: "mcr.microsoft.com/azureiotedge-agent:1.0.8-rc1"_' $(RFSDIR)/etc/iotedge/config.yaml
 endif
 
 
-- 
2.21.0

